src/components/SiteHeader.vue

```diff
@@ -150,23 +117,127 @@
                   'flex',
                   'justify-center',
                   'items-center',
+                  'rounded-full',
+                  'min-w-[20px]',
+                  'min-h-[20px]',
+                  'ml-[-10px]',
+                  'mb-[-10px]',
+                  'px-[4px]',
+                  'py-[5px]',
+                  'pointer-events-none',
                   'bg-shade-gray',
                   { 'bg-danger': getNotificationCount },
-                  'rounded-full',
-                  'min-w-[24px]',
-                  'px-[8px]',
-                  'py-[4px]',
                 ]"
               >
-                <div class="text-white text-[10px]">
+                <div class="text-white text-[10px] leading-[1em]">
                   {{ formattedNotificationCount }}
                 </div>
               </div>
             </template>
           </MenuItem>
         </MenuList>
       </Dropdown>
+
+      {{ /* phone version */ }}
+      <ButtonV2
+        v-if="!getAddress"
+        class="laptop:hidden"
+        preset="plain"
+        @click="handleOpenSider"
+      >
+        <IconNav />
+      </ButtonV2>
+      <Identity
+        v-if="getAddress"
+        class="cursor-pointer laptop:!hidden ml-[12px]"
+        :avatar-url="walletUserAvatar"
+        :avatar-size="42"
+        :is-avatar-outlined="isWalletUserCivicLiker"
+        @click="handleOpenSider"
+      />
     </div>
+
+    <SiteMenuSider v-if="isShowSider" @close="isShowSider = false">
```

Comment:
`<SiteMenuForMobile />` ?? not sure what is sider

Reply:
typo.... I mean slider ..... 🤣🤣🤣

Reply:
But normally Slider means this component
<img width="627" alt="image" src="https://github.com/user-attachments/assets/89658647-9a1c-4c3a-8488-21e1560be5a0">

I think `<SiteMenuForMobile />` is pretty straightforward
toggle with `isShowMobileMenu`

Reply:
on second thought

`<SiteMenuPopover v-if="isShowPopover" />`?

Reply:
[🎨 Rename mobile menu](https://github.com/likecoin/liker-land/pull/1724/commits/b622c36df27143b04969f2eb0a8fafa68454265d)